### PR TITLE
docs: add quickstart documentation

### PR DESCRIPTION
**Pull request by Meghajit**
This is the first step towards creating a better quickstart documentation. Contains more step by step details on how to run dagger locally.

In the future, we will add more examples.

### CODE REVIEW

File: quickstart/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Dagger Quickstart
```
Comment (prakharmathur82) [inline quickstart/README.md:1]: can we move this at https://github.com/odpf/dagger/tree/doc/add-quickstart-documentation/docs/docs/guides?

File: quickstart/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+4. You have **protobuf** installed: We will use protobuf to push messages encoded in protobuf format to Kafka topic. You can follow the installation steps for MacOS [here](https://formulae.brew.sh/formula/protobuf). For other OS, please download the corresponding release from [here](https://github.com/protocolbuffers/protobuf/releases). Please note, this quickstart has been written to work with[ 3.17.3](https://github.com/protocolbuffers/protobuf/releases/tag/v3.17.3) of protobuf. Compatibility with other versions is unknown.
+5. You have **Python 2.7+** and **simple-http-server** installed: We will use Python along with simple-http-server to spin up a mock Stencil server which can serve the proto descriptors to Dagger. To install **simple-http-server**, please follow these [installation steps](https://pypi.org/project/simple-http-server/).
+
+## Quickstart
+
+1. Clone dagger repository into you local. Checkout into the branch [`doc/add-quickstart-documentation`](https://github.com/odpf/dagger/tree/doc/add-quickstart-documentation). (You should see a folder at the top level called `quickstart` containing this README.)
```
Comment (prakharmathur82) [inline quickstart/README.md:13]: we can change the path now

**Review by prakharmathur82 — CHANGES_REQUESTED** [review]

File: quickstart/README.md (hunk 2 above)
Comment (Meghajit) [reply quickstart/README.md]: Done

File: quickstart/README.md (hunk 1 above)
Comment (Meghajit) [reply quickstart/README.md]: Moved.
Have also linked the quickstart in the main README

**Review by prakharmathur82 — APPROVED** [review]

**Pull request merged**